README.md

```diff
@@ -100,12 +100,12 @@ jobs:
         uses: neofinancial/ticket-check-action@v1
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          ticketLink: 'https://app.clubhouse.io/:org/story/%ticketNumber%'
-          ticketPrefix: 'CH-'
-          titleRegex: '^(CH)(-?)(?<ticketNumber>\d{3,})'
-          branchRegex: '^(CH)(-?)(?<ticketNumber>\d{3,})'
-          bodyRegex: '(CH)(-?)(?<ticketNumber>\d{3,})'
-          bodyURLRegex: 'http(s?):\/\/(app.clubhouse.io)(\/:org)(\/story)\/(?<ticketNumber>\d+)'
+          ticketLink: 'https://app.shortcut.com/:org/story/%ticketNumber%'
+          ticketPrefix: 'sc-'
```

Comment:
We should probably stick with an uppercase prefix. We could also make the regex case insensitive if it's not already.

Reply:
Makes sense. TicketPrefix is a bit confusing but as far as I know, it is only used to update the PR title in case the titleRegex fails (which I think solves the case sensitive issue).